content/applications/productivity/iot/config/connect.rst

```diff
@@ -28,54 +31,78 @@ Ethernet Connection
+   Recent changes in modern web browsers forced us to modify the connection wizard. If the screen is
+   different from the screenshots, make sure that the ``iot_pairing`` module is installed.
+
+..
+   _This module doesn't exist any longer.
 
 WiFi Connection
 ===============
 
 #. Power on the IoT Box
 
-#. Copy the token
+#. Copy the token from the WiFi connection section in IoT module of the Odoo database.
 
    .. image:: connect/connect05.png
+      :align: center
+      :alt: Pairing Token for a WiFi connection.
 
-#. Connect to the IoT Box WiFi Network (make sure there is no ethernet
-   cable plugged in your computer).
+#. Connect to the IoT Box WiFi Network (make sure there is no ethernet cable plugged in the
+   computer).
 
    .. image:: connect/connect06.png
+      :align: center
+      :alt: WiFi networks available on the computer.
+
+..
+   _Image needs to be updated.
 
-#. You will be redirected to the IoT Box Homepage (if it doesn't work,
-   connect to the IP address of the box). Give a name to your IoT Box (not
-   required) and paste the token, then click on next.
+#. Upon connecting to the IoT WiFi a browser will automatically redirect to the IoT Box Homepage (if
+   it doesn't work, connect to the IP address of the box). The default address of the IoT box is
+   `localhost:8069 <http://localhost:8069/>`. Give a name to the IoT Box (not required), paste the
+   previously copied token, and then click on next.
 
    .. image:: connect/connect07.png
+      :align: center
+      :alt: A connection has been made with the IoT box and Odoo database.
 
    .. tip::
-      If you are on Runbot, do not forget to add the -all or -base in the
+      If the configuration is on Runbot, do not forget to add the -all or -base in the
       token (e.g. this token
       **http://375228-saas-11-5-iot-f3f920.runbot16.odoo.com\|4957098401**
       should become
       **http://375228-saas-11-5-iot-f3f920-all.runbot16.odoo.com\|4957098401**).
 
-#. Choose the WiFi network you want to connect with (enter the password
-   if there is one) and click on Submit. Wait a few seconds before being
-   redirected to your database.
+..
+  _Confirm the web address prior to entering the token.
+
+#. Choose the WiFi network that the IoT box will connect with (enter the password if there is one)
+   and click on Submit. Wait a few seconds before being redirected to the database. The computer may
+   need to be manually connected back to WiFi.
 
    .. image:: connect/connect08.png
+      :align: center
+      :alt: Configuring the WiFi for the IoT box.
 
-You should now see the IoT Box.
+The IoT box should appear in the IoT module of the Odoo database.
 
 .. image:: connect/connect09.png
    :align: center
+   :alt: The IoT box has been successfully configured on the Odoo database.
 
 IoT Box Schema
 ==============
 
 .. image:: connect/connect10.png
-   :align: center
+      :align: center
+      :alt: Odoo IoT box schema with labels.
```

Comment:
this schema is actually not really accurate.
The schema on this one is for the raspberry pi 3b that we used to use before. However the more recent model that Odoo ship is the model 4 which is a bit different.
The main difference (if we ignore the performances related one as not really relevant here) is the fact that:
 - HDMI -> 2 micro-hdmi
 - Micro USB power -> USB-C power
 - USBs & Ethernet Switch place

Might be a good occasion to mention the differences between both models

Reply:
@lse-odoo excellent catch, I've reworked the Schema image, can you verify my layout matches the new Raspberry Pi 4?

Reply:
@tiku-odoo 
It's correct :) Maybe we can remove the "audio" label as to my knowledge no device use it.
However, I wasn't expecting that you remove the previous one. As both raspberry pi 3b and 4 can be used, it would be relevant in my opinion to keep both schemas and clearly indicate on them what models they are. As the different models are sometimes referred in other part of the documentation like at:
IoT Box Schema.

Note that you can use `.. figure` instead of `.. image ` if you want to add a label underneath

Reply:
Great suggestion; I've added the schema for the Raspberry Pi 3 and 4 into the doc. The status lights are in the incorrect place on the original schema for the Pi 3. Can you confirm?

Reply:
@tiku-odoo based on the picture that I saw online it looks to be on the corner close to the power source on model 2 and 4 :)